staging/src/k8s.io/client-go/tools/auth/exec/exec_test.go

```diff
@@ -0,0 +1,189 @@
+package exec
+
+import (
+	"strings"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/diff"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientauthenticationv1alpha1 "k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1"
+	clientauthenticationv1beta1 "k8s.io/client-go/pkg/apis/clientauthentication/v1beta1"
+	"k8s.io/client-go/rest"
+)
+
+// restInfo holds the rest.Client fields that we care about for test assertions.
+type restInfo struct {
+	host            string
+	tlsClientConfig rest.TLSClientConfig
+	proxyURL        string
+}
+
+func init() {
+	utilruntime.Must(corev1.AddToScheme(scheme))
+}
+
+func TestLoadExecCredential(t *testing.T) {
```

Comment:
I manually tested `LoadExecCredentialFromEnv` using a one-off exec plugin and a `kubectl` built from source. Should this manual test be converted to an automatic test and checked into the repo somewhere?

Reply:
I think the individual unit tests are probably sufficient